homeassistant/components/sensibo/services.yaml

```diff
@@ -16,3 +16,18 @@ assume_state:
           options:
             - "on"
             - "off"
+horizontal_swing:
+  name: Horizontal swing
+  description: Set Sensibo device horizontal swing
+  target:
+    entity:
+      integration: sensibo
+      domain: climate
+  fields:
+    state:
+      name: State
+      description: State to set.
+      required: true
+      example: "stopped"
```

Comment:
Are there only two states?  Can this be a switch?

Reply:
No, there is a whole set of options and it can be different depending on model so can't set it fixed.
Example for mine:
```json
"horizontalSwing": [
                "stopped",
                "fixedLeft",
                "fixedCenterLeft",
                "fixedCenter",
                "fixedCenterRight",
                "fixedRight",
                "fixedLeftRight",
                "rangeCenter",
                "rangeFull"
              ]
```

Reply:
This should probably be a `select` entity in that case

Reply:
Can you make a dynamic list for such selector? Any example somewhere?

Reply:
https://developers.home-assistant.io/docs/core/entity/select?_highlight=select

Does the list of options ever change or are they always the same for the device?

Reply:
If you have a diagnostics file from one with `horizontalSwing` it might help give better context

Reply:
I got one from a different PR who has a list which is completely different than mine. Actually it's also different from swing but here in the services it's free text (wonder why it's not populated from the property but maybe this selector is pretty new?) hence why I also made this free text.
I will look at this select entity.

Reply:
As I now did a select here for horizontal swing would it make sense to make one for swing as well given it's similar or we keep ourselves to the already existing service call (even though there you can't select, you have to type the state you want)?

Reply:
> As I now did a select here for horizontal swing would it make sense to make one for swing as well given it's similar or we keep ourselves to the already existing service call (even though there you can't select, you have to type the state you want)?

We should use the built-in climate entity when possible.  We should only create an additional select when the climate entity cannot represent the state.